pyuvdata/uvfits.py

```diff
@@ -104,12 +104,19 @@ def _get_parameter_data(self, vis_hdu):
                                  * const.c.to('m/s').value).T
 
         if 'INTTIM' in vis_hdu.data.parnames:
-            self.integration_time = float(vis_hdu.data.par('INTTIM')[0])
+            inttim = vis_hdu.data.par('INTTIM')
+            if len(inttim) == 1:
+                # assume that all integration times in the file are the same
```

Comment:
I think I tried this originally, but some of the test files were non-compliant and only have an array that's a single entry long (or at least, not Nblts long). I can inspect these files with a FITS reader to see what's actually going on.

Reply:
If you know which files were problematic I'd be happy to look at them too. I actually noticed this because it wasn’t covered in the tests, so it seems like we aren't hitting this clause in our existing tests.

Reply:
I just tried running the tests without this if statement, and I didn't have any issues. I must have been misremembering a different issue I was having. I'll remove this clause, thanks for finding it!